### PR TITLE
Update datocms.json

### PR DESCRIPTION
**Pull request by matjack1**
Hello,

just an update on new DatoCMS's features! :)

Thank you!

### CODE REVIEW

**Comment by Jotschi** [comment]
@matjack1 Thanks for the PR. Please provide links to the docs of the mentioned features. Otherwise I can't verify that the claims are valid. Thanks.

**Comment by matjack1** [comment]
Sure @Jotschi !

- for the on-premise installation is something that we offer in the enterprise, it's not publicly stated, is it a problem?
- geolocation: https://www.datocms.com/docs/content-modelling there's a "Geolocation" field and we can search by point and range
- we have Imgix integrated as asset manipulation and they support fingerprinting: https://docs.imgix.com/best-practices/fingerprinting-images-to-improve-page-load-speed
- JS SDK: https://github.com/datocms/js-datocms-client/
- plugin system: https://www.datocms.com/plugins
- customisable UI: the plugins are meant for that
- custom field types: https://www.datocms.com/docs/content-modelling
- doc level permission: https://www.datocms.com/docs/general-concepts/roles-and-permission-system you can see that you can specify "all records"/specify record ids
- Oauth: again this is something that we can setup custom for certain users, it's not publicly available for everyone
- content trees: https://www.datocms.com/docs/content-modelling/trees/
- tagging: we have a plugin for that: https://www.datocms.com/plugins/i/datocms-plugin-tag-editor/
- GDPR: https://www.datocms.com/legal/gdpr
- conflict detection: https://www.datocms.com/docs/general-concepts/collaboration-features
- content scheduling: https://www.datocms.com/changelog/scheduled-publishing/

if you need more info please let me know!

File: datocms.json [hunk 1]
```diff
@@ -47,39 +47,38 @@
...
-    "Customizable UI": "No",
-    "Custom Field Types": "No",
+    "Plugin System": "Yes",
+    "Customizable UI": "Yes",
+    "Custom Field Types": "Yes",
     "User Management": "Yes",
     "Role Based Permissions": "Yes",
-    "Document Level Permissions": "No",
-    "OAuth 2.0 Support": "No",
+    "Document Level Permissions": "Yes",
```
[the remark this answers is not in the document]
Comment (Jotschi) [reply datocms.json]: I did not see from the docs that it is possible to reference record ids. Did I miss that part?

File: datocms.json [hunk 2]
```diff
@@ -47,39 +47,38 @@
...
+    "OAuth 2.0 Support": "Yes",
```
[the remark this answers is not in the document]
Comment (matjack1) [reply datocms.json]: OK, that's fine :)

I've reverted it.

File: datocms.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (matjack1) [reply datocms.json]: You are right, I was thinking permission per model. If you mean by record instead we cannot do that. I've amended that, sorry.

File: datocms.json [hunk 3]
```diff
@@ -47,39 +47,38 @@
...
     "Web Hooks": "Yes",
     "Eventbus": "No",
     "Bulk Import": "Yes",
     "Client Side Forms": "No",
-    "Plugin System": "No",
-    "Customizable UI": "No",
-    "Custom Field Types": "No",
+    "Plugin System": "Yes",
+    "Customizable UI": "Yes",
+    "Custom Field Types": "Yes",
```
[the remark this answers is not in the document]
Comment (matjack1) [reply datocms.json]: do you want me to delete this line? Shall we do maybe in a separate PR for all the entries maybe?

[the remark this answers is not in the document]
Comment (Jotschi) [reply datocms.json]: I have now removed the feature from the list. No need to take action.

**Comment by Jotschi** [comment]
@matjack1 Thanks for the PR. I have now manually merge the changes.

**Pull request closed without merging**